griptape/tools/email_client/tool.py

```diff
@@ -160,27 +167,43 @@ def send(self, params: dict) -> InfoArtifact | ErrorArtifact:
         smtp_host = self.smtp_host
         smtp_port = int(self.smtp_port)
 
-        to_email = input_values["to"]
-        subject = input_values["subject"]
```

Comment:
Why remove those? You have them in `GoogleGmailClient`.

Reply:
just thought its cleaner since im using it here 

```
msg["To"] = input_values["to"]
msg["Subject"] = input_values["subject"]
```

Reply:
But for consistency, will put it back!